cmd/kubeadm/app/util/version.go

```diff
@@ -184,7 +187,15 @@ func fetchFromURL(url string, timeout time.Duration) (string, error) {
 func kubeadmVersion(info string) (string, error) {
 	v, err := versionutil.ParseSemantic(info)
 	if err != nil {
-		return "", pkgerrors.Wrap(err, "kubeadm version error")
+		// In case we run `go test` in git tree the version comes from
+		// unmodified pkg/version/base.go. Then we'd better fall back
+		// to a predefined but meaningful constant.
+		if strings.HasPrefix(info, "v0.0.0-master") {
+			klog.Warningf("Can't use in-tree client version. Fall back to minimum control plane version")
+			v = constants.MinimumControlPlaneVersion
```

Comment:
it's not a good fallback because it's MINOR-1.

Reply:
@rojkov I support the opinion of @neolit123 . We have to be pointing at the current version.
Luckily, @bart0sh is working on a series of PRs, that will introduce some constants, possibly for the current version too. You can sync together for a constant to be used in this PR, but you'll probably have to wait a bit for Ed's PRs.